hnn_core/tests/test_network.py

```diff
@@ -252,18 +333,17 @@ def test_network():
                        match="'times' is an np.ndarray of simulation times"):
         _ = CellResponse(times='blah')
 
-    # Assert that all external drives are initialized
-    # Assumes legacy mode where cell-specific drives create artificial cells
-    # for all network cells regardless of connectivity
-    n_evoked_sources = 3 * net._n_cells
-    n_pois_sources = net._n_cells
-    n_gaus_sources = net._n_cells
-    n_bursty_sources = (net.external_drives['bursty1']['n_drive_cells'] +
-                        net.external_drives['bursty2']['n_drive_cells'])
+    # Check that all external drives are initialized with the expected amount
+    # of artificial cells assuming legacy_mode=False (i.e., dependent on
+    # drive targets).
+    prox_targets = net._n_cells
+    dist_targets = net._N_pyr_x * net._N_pyr_x * 2 + 35  # 35 L2/3 basket cells
```

Comment:
can we use `len(net.gid_ranges['basket_L5'])` to avoid hardcoding 35?

Reply:
Sure, that's probably clearer.

Reply:
Fixed in https://github.com/jonescompneurolab/hnn-core/pull/431/commits/b9de47a9028aed20f0b08667e771fe17873cac6d!